packages/taquito-michelson-encoder/src/tokens/or.ts

```diff
@@ -13,6 +20,17 @@ export class OrTokenDecodingError extends TaquitoError {
   }
 }
 
+/**
+ *  @category Error
+ *  @description Error that indicates a failure happening when parsing encoding/executing an OrToken
```

Comment:
suggestion only, from this description seems where throw OrTokenDecodingError can be replaced instead. (complying with structure of other tokens files)